spec/ddtrace/contrib/sinatra/activerecord_spec.rb

```diff
@@ -20,7 +20,7 @@
   before(:each) do
     Datadog.configure do |c|
       c.use :sinatra, options
-      c.use :active_record
+      c.use :active_record, options
```

Comment:
This test started to fail because it was not configuring a test tracer, using the global tracer instead.
The global tracer was missing a few ActiveRecord bootstrap queries (`PRAGMA` queries). We assumed they were never present, even though these queries do show in a real application.